### PR TITLE
[14.0][FIX] resource_booking: Force to send calendar event author notification.

### PR DESCRIPTION
**Pull request by victoralmau**
Force to send calendar event author notification (https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_thread.py#L2470).

Steps to reproduce:

- Create a booking
- Set a manual assigned
- Share + confirm (with portal)
- User has been notified

Please @pedrobaeza can you review it?

@Tecnativa TT40302

### CODE REVIEW

File: resource_booking/models/resource_booking.py [hunk 1]
```diff
@@ -532,29 +532,43 @@ def name_get(self):
             new.append((id_, name))
         return new
 
+    def _get_combination_from_updated_values(self, updated_values):
```
Comment (pedrobaeza) [inline resource_booking/models/resource_booking.py]: Why extracting a method? Is it used in another part?

File: resource_booking/models/resource_booking.py [hunk 2]
```diff
@@ -532,29 +532,43 @@ def name_get(self):
...
-                    partner.id,
-                    default_subtype_ids,
-                    "resource_booking.message_combination_assigned"
-                    if partner != self.env.user.partner_id
-                    else False,
+        combination = self._get_combination_from_updated_values(updated_values)
+        if combination:
+            resource_partners = combination.mapped("resource_ids.user_id.partner_id")
+            for partner in resource_partners.filtered("active"):
+                result.append(
```
Comment (pedrobaeza) [inline resource_booking/models/resource_booking.py]: Doing this they receive the booking details? What if they aren't subscribed by default to notes?

**Review by pedrobaeza — COMMENTED** [review]
Please check in v13, that the notifications were done by calendar module standard calendar event mail template and mechanisms

**Comment by victoralmau** [comment]
> Please check in v13, that the notifications were done by calendar module standard calendar event mail template and mechanisms

Sorry, but if I try v13 I don't get an email in the booking confirmation either.

**Comment by pedrobaeza** [comment]
I was getting it from calendar.event. Probably the problem is that you are confirming the booking from your own user. Try to book it from the portal in an incognito window.

**Comment by victoralmau** [comment]
Ok, it has been checked and the necessary fix is now applied.

**Review by pedrobaeza — APPROVED** [review]
Tested in a real environment, and working as expected. Fast-tracking this to solve the problem:

/ocabot merge patch

**Pull request merged**